fctesting/utils.go

```diff
@@ -43,3 +46,24 @@ func RequiresRoot(t testing.TB) {
 			rootDisableEnvName)
 	}
 }
+
+func newLogger(t testing.TB) *log.Logger {
+	str := os.Getenv(logLevelEnvName)
+	if str == "" {
+		return log.New()
+	}
+
+	logLevel, err := log.ParseLevel(str)
+	if err != nil {
+		t.Fatalf("Failed to parse '%s' as Log Level: %v", str, err)
```

Comment:
nit: you can use `%q` to single quote wrap a string.

Reply:
Didn't know that. Will fix!